bin/http-server

```diff
@@ -75,6 +82,22 @@ var port = argv.p || argv.port || parseInt(process.env.PORT, 10),
     version = argv.v || argv.version,
     logger;
 
+
+if (nopanic){
+
+    process.on('error', (e)=> {
+      let etime = new Date().toISOString().replace(/T/, ' ').replace(/\..+/, '')   
+      console.log(colors.green(etime)) 
+      console.log(colors.red("Fatal error: ")+ e.code + ": "+e.message)
+      let fname = ".httpserver-"+etime.split(" ")[0]+"-"+etime.split(" ")[1]
```

Comment:
For the log file, could we end it with `.log` to make it clear what the file is?